### DISCUSSION TITLE
Error on gradle build, A problem occurred configuring root project 'pack'.

### DISCUSSION THREAD

**Discussion by chillnook** [FabricMC/fabric-api#1447] [Mod Dev Support] [answered]
Never properly solved, delete maybe?

**Comment by liach** [comment]
Hmm, try `./gradlew --stop` and then `./gradlew wrapper --gradle-version 7.0.2`, finally `./gradlew clean genSources --refresh-dependencies` and see if the problem still persists.

**Reply by chillnook** [reply]
As soon as I do /gradlew clean genSources --refresh-dependencies the same Could not resolve net issue comes back.

**Comment by liach** [comment]
Try run all gradle tasks with `-S` flag in the end and post their crash logs.

**Comment by Martmists-GH** [comment, marked as answer]
Update loom to a newer version

**Comment by chillnook** [comment]
Reinstalling Windows seemed to do the trick, thanks for your help everyone.
